front/src/reducers/editor.ts

```diff
@@ -53,13 +54,14 @@ export function selectLayers(
 // Verify if the data model definition is already loaded.
 // If not we do it and store it in the state
 //
-const LOAD_DATA_MODEL = 'editor/LOAD_DATA_MODEL';
+export const LOAD_DATA_MODEL = 'editor/LOAD_DATA_MODEL';
```

Comment:
No need to export this
```suggestion
const LOAD_DATA_MODEL = 'editor/LOAD_DATA_MODEL';
```